avahi-service/revpi.service

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" standalone='no'?>
```

Comment:
Yesterday you told us, you would NOT use the name `revpi.service`. I would suggest to rename the file to `revpi-ssh.service`. The prefix could be important and indicates that it is our file.

Other files form web status could be `revpi-http.service` or things like that.